src/pycolmap/scene/image.cc

```diff
@@ -52,80 +65,94 @@ std::shared_ptr<Image> MakeImage(const std::string& name,
   return image;
 }
 
+template <typename T>
+std::shared_ptr<Image> MakeImage(Camera* camera,
+                                 const std::string& name,
+                                 const std::vector<T>& points2D,
+                                 const Rigid3d& cam_from_world,
+                                 image_t image_id) {
+  auto image = std::make_shared<Image>(camera);
+  image->SetName(name);
+  image->SetPoints2D(points2D);
+  image->CamFromWorld() = cam_from_world;
+  image->SetImageId(image_id);
+  return image;
+}
```

Comment:
```suggestion
std::shared_ptr<Image> MakeImage(Camera* camera,
                                 std::string name,
                                 std::vector<T> points2D,
                                 Rigid3d cam_from_world,
                                 image_t image_id) {
  auto image = std::make_shared<Image>(camera);
  image->SetName(std::move(name));
  image->SetPoints2D(std::move(points2D));
  image->CamFromWorld() = std::move(cam_from_world);
  image->SetImageId(image_id);
  return image;
}
```

Reply:
This implementation was kept consistent with https://github.com/colmap/colmap/blob/main/src/pycolmap/scene/image.cc#L39. Also May I ask what the issue is for the const reference here?